libraries/objectDefaultFiles/object.js

```diff
@@ -2151,6 +2158,36 @@
                 }
             })
         }
+        
+        this.toggleMeasureMode = function(boolean) {
+            postDataToParent({
+                toggleMeasureMode: boolean
+            });
+        }
+
+        this.toggleCrossRotation = function(boolean) {
+            postDataToParent({
+                toggleCrossRotation: boolean
+            });
+        }
+        
+        this.toggleCloseLoop = function(boolean) {
```

Comment:
I'm guessing this API relates to changing the appearance of the spatial cursor, but that's only because I know what you've been working on – can we rename all the APIs that affect the spatial cursor appearance to be a bit more clear that they're affecting the cursor appearance?